enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,1319 @@
+    status: "True"
+    type: DeploymentReplicasAllAvailable
+  - lastTransitionTime: "2024-04-10T00:20:50Z"
+    message: Deployment is not actively rolling out
+    reason: DeploymentNotRollingOut
+    status: "False"
+    type: DeploymentRollingOut
+  - lastTransitionTime: "2024-04-10T00:20:17Z"
+    message: The endpoint publishing strategy supports a managed load balancer
+    reason: WantedByEndpointPublishingStrategy
+    status: "True"
+    type: LoadBalancerManaged
+  - lastTransitionTime: "2024-04-10T00:20:22Z"
+    message: The LoadBalancer service is provisioned
+    reason: LoadBalancerProvisioned
+    status: "True"
+    type: LoadBalancerReady
+  - lastTransitionTime: "2024-04-10T00:20:17Z"
+    message: LoadBalancer is not progressing
+    reason: LoadBalancerNotProgressing
+    status: "False"
+    type: LoadBalancerProgressing
+  - lastTransitionTime: "2024-04-10T00:20:17Z"
+    message: DNS management is supported and zones are specified in the cluster DNS
+      config.
+    reason: Normal
+    status: "True"
+    type: DNSManaged
+  - lastTransitionTime: "2024-04-10T00:20:22Z"
+    message: The record is provisioned in all reported zones.
+    reason: NoFailedZones
+    status: "True"
+    type: DNSReady
+  - lastTransitionTime: "2024-04-10T00:20:50Z"
+    status: "True"
+    type: Available
+  - lastTransitionTime: "2024-04-10T00:20:50Z"
+    status: "False"
+    type: Progressing
+  - lastTransitionTime: "2024-04-10T00:20:50Z"
+    status: "False"
+    type: Degraded
+  - lastTransitionTime: "2024-04-10T00:20:17Z"
+    message: IngressController is upgradeable.
+    reason: Upgradeable
+    status: "True"
+    type: Upgradeable
+  - lastTransitionTime: "2024-04-10T00:20:17Z"
+    message: No evaluation condition is detected.
+    reason: NoEvaluationCondition
+    status: "False"
+    type: EvaluationConditionsDetected
+  domain: apps1.apps.misalunkeip16.devcluster.openshift.com
+  endpointPublishingStrategy:
+    loadBalancer:
+      dnsManagementPolicy: Managed
+      providerParameters:
+        aws:
+          networkLoadBalancer:
+            eipAllocations:
+            - eipalloc-0956fea34de4cb7ab
+            - eipalloc-0e9a3077a70de050a
+            - eipalloc-0b69fc4691f54cdd0
+            - eipalloc-01e6ba6cbba1a391b
+            - eipalloc-0e242df173f906112
+          type: NLB
+        type: AWS
+      scope: External
+    type: LoadBalancerService
+  observedGeneration: 2
+  selector: ingresscontroller.operator.openshift.io/deployment-ingresscontroller=test
+  tlsProfile:
+    ciphers:
+    - ECDHE-ECDSA-AES128-GCM-SHA256
+    - ECDHE-RSA-AES128-GCM-SHA256
+    - ECDHE-ECDSA-AES256-GCM-SHA384
+    - ECDHE-RSA-AES256-GCM-SHA384
+    - ECDHE-ECDSA-CHACHA20-POLY1305
+    - ECDHE-RSA-CHACHA20-POLY1305
+    - DHE-RSA-AES128-GCM-SHA256
+    - DHE-RSA-AES256-GCM-SHA384
+    - TLS_AES_128_GCM_SHA256
+    - TLS_AES_256_GCM_SHA384
+    - TLS_CHACHA20_POLY1305_SHA256
+    minTLSVersion: VersionTLS12
+```
+
```

Comment:
The router-test service will now show the updated  `service.beta.kubernetes.io/aws-load-balancer-eip-allocations` annotation values.

Reply:
ok

Reply:
added